Documentation/ceph-cluster-crd.md

```diff
@@ -948,6 +948,14 @@ Finally, you can simply execute the script like this from a machine that has acc
 ```console
 bash cluster/examples/kubernetes/ceph/import-external-cluster.sh
 ```
+Choose the namespace carefully, if you have an existing cluster managed by Rook, you have likely already injected `common.yaml`.
```

Comment:
We only need to create common-external.yaml if there is a different namespace than `rook-ceph` or if you want to run both a local cluster and an external cluster, right? But I'm still not clear from this section. The main scenario should be to only connect to the external cluster. It should be uncommon to have a local cluster at the same time except for testing purposes. Let's focus this doc on the scenario of only having an external cluster. How does that change this paragraph?

Reply:
I am not sure if it is uncommon . For instance, we have a storage cluster that we have external ceph on , along with internal ceph.  But I agree this maybe not so common. If you feel so I will remove it and add only the Storage yaml example.